### ISSUE TITLE
additional auxiliary loss for moe?

### ISSUE THREAD

**Issue by maxin-cn** [FreedomIntelligence/LongLLaVA#12]
Thanks for the previous responses! The LongLLaVA model utilizes the Mixture of Experts (MoE), but it seems that MoE has a "winner-takes-all" issue (i.e., Expert Balancing). Was any additional regularization applied to MoE during model training?

**Comment by wangxidong06** [comment]
Yes we use router_aux_loss_coef with [1e-3](https://github.com/FreedomIntelligence/LongLLaVA/blob/main/llava/model/language_model/Jamba/config.json#L39) and employ modeling.py in [here](https://github.com/FreedomIntelligence/LongLLaVA/blob/main/llava/model/language_model/Jamba/modeling_jamba.py#L1892)

**Comment by maxin-cn** [comment]
Thanks for your reply!
